common/customizable_localization/wc_lifestyles_custom_loc.txt

```diff
@@ -2,48 +2,48 @@
 	type = character
 
 	text = {
```

Comment:
I was thinking. Why approved doctrines don't add magic traits to virtues? They will have the same opinion functionality plus they will get piety.

Reply:
done